example_1.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Example 1</title>
+  <meta charset="utf-8">
+  <script>
+
+    var names = "Pedro Picapiedra; Pablo Marmol; Roger Federer; Rafael Nadal; Andy Murray";
+
+    var exit = ["Original string: "+names];
```

Comment:
pon espacios entre operadores:
```suggestion
    var exit = [ "Original string: " + names ];
```